DynamicActiveRecord.php

```diff
@@ -234,16 +234,7 @@ function ($matches) {
      */
     public function __get($name)
     {
-        $value = null;
-        try {
-            $value = parent::__get($name);
```

Comment:
okay, did not know that. the only problem here is that this hides the exception thrown on unknown property if it is really not existing.